opentracing-shim/src/main/java/io/opentelemetry/opentracingshim/OpenTracingPropagators.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.opentracingshim;
+
+import io.opentelemetry.context.propagation.TextMapPropagator;
+
+/**
+ * Container for {@link io.opentracing.propagation.Format.Builtin#TEXT_MAP} and {@link
+ * io.opentracing.propagation.Format.Builtin#HTTP_HEADERS} format propagators.
+ */
+public class OpenTracingPropagators {
+  private final TextMapPropagator textMapPropagator;
+  private final TextMapPropagator httpHeadersPropagator;
+
+  public TextMapPropagator textMapPropagator() {
+    return textMapPropagator;
+  }
+
+  public TextMapPropagator httpHeadersPropagator() {
+    return httpHeadersPropagator;
+  }
+
+  public OpenTracingPropagators(
```

Comment:
Actually, let's hide this one, as we already are providing the builder.

Reply:
done